app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(file_names: str) -> None:
+    command, original, copy = file_names.split()
+    if original == copy:
+        exit(0)
+    if command == "cp":
```

Comment:
the same, no need exit here, make one condition form this both with `and` operator